frontend/src/pages/EventSignInOptions/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router';
+import { Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+import useStyles from './styles';
+
+import * as ROUTES from '@Constants/routes';
+import { getEventById } from '@Services/EventService';
+import { EventResponse } from '@Services/api/models';
+
+interface EventID {
+  id: string;
+}
+
+function EventSignInOptions(): JSX.Element {
+  const { id } = useParams<EventID>();
+  const eventId = parseInt(id, 10);
+  const [eventInfo, setEventInfo] = useState<EventResponse | null>(null);
+
+  useEffect(() => {
+    const getEvent = async () => {
+      const eventResponse = await getEventById(eventId);
+      setEventInfo(eventResponse);
+    };
+
+    getEvent();
+  }, [eventId]);
+
+  const signInURL = eventInfo == null ? ' ' : eventInfo.signInURL;
+
+  const classes = useStyles();
```

Comment:
Lines 30 and 32 should be at the top of the function definition since they are assignments